template/package.json

```diff
@@ -14,7 +14,8 @@
   },
   "dependencies": {
     "vue": "^2.2.2"{{#router}},
```

Comment:
You can end up with a trailing comma here if you don't include `vuex` or `vue-router`.